source/manual/kubernetes-infrastructure.html.md

```diff
@@ -7,93 +7,175 @@ parent: "/manual.html"
 
-## Dashboards and alerting
+If you try to open a shell on Router, it will fail:
 
-- The detailed Sidekiq dashboard is not yet available for EKS (the [old one](https://sidekiq-monitoring.integration.govuk.digital/publishing-api/queues "‌") still looks at EC2) but you can see queue lengths and age-of-oldest-job on the [Sidekiq Grafana dashboard](https://grafana.eks.production.govuk.digital/d/2Yy8PzmVk "‌").
-- Dashboards are on [https://grafana.eks.production.govuk.digital/](https://grafana.eks.production.govuk.digital/?orgId=1 "‌") and no longer require VPN. Don’t get confused by the [old Grafana](https://grafana.production.govuk.digital/ "‌"), which will show mostly empty graphs about the old EC2 setup.
-- Most Icinga alerts are irrelevant now, except for:
-    - “scheduled publications in Whitehall not queued” / “overdue publications in Whitehall”
-    - “Travel Advice email alert check” / “Medical Safety Email alert check”
-    - Anything to do with crawler/mirrors
-    - Anything to do with Licensify
+```sh
+$ k exec -it deploy/router -- bash                                                               
+Defaulted container "app" out of: app, nginx
+error: Internal error occurred: error executing command in container: failed to exec in container: failed to start exec "e870a2214da6273408c75773234b09c245210d3f1dff3b200c096f24c7259f7f": OCI runtime exec failed: exec failed: unable to start container process: exec: "bash": executable file not found in $PATH: unknown
+```
 
-## Testing EKS access
+This is because [Router's image is `FROM scratch`](https://github.com/alphagov/router/blob/9797473edbbcbb5085fdca006bec7f6b1552f4e6/Dockerfile#L7), so bash isn't available. You'll need to exec into the nginx container instead (which does have a shell):
 
-See [testing your eks cluster access in the GOV.UK Kubernetes platform user documentation](https://govuk-k8s-user-docs.publishing.service.gov.uk/get-started/access-eks-cluster/#test-your-access)
+```sh
+k exec -it deploy/router -c nginx -- bash
+```
 
-## Kubernetes cheat sheet
+## Run a rake task
 
-[Official k8s cheat sheet](https://kubernetes.io/docs/reference/kubectl/cheatsheet/)
+See [Run a Rake task on EKS](https://docs.publishing.service.gov.uk/manual/running-rake-tasks.html#run-a-rake-task-on-eks)
 
-## What’s staying and going?
+Here's an idempotent rake task you can run to try one out:
 
-[Document outlining what changes are happening to GOV.UK infrastructure](https://docs.google.com/document/d/1R8C3BtvhqTXEga4C3_KxTopjWuYVbiEgKiikTyRXXiA/edit#)
+```sh
+k exec deploy/email-alert-api -- rake 'support:view_emails[your.email@digital.cabinet-office.gov.uk]'
+```
+
+Note that we haven't specified the `-it` flags in the command above.
+To find out what they do, run `k exec --help`.
+
+### Testing changes to a rake task
+
+For special router publisher as an example:
+
+1. Deploy your branch (see above)
+2. Run the following command(s):
```

Comment:
Oh I didn't add this section, I didn't review its contents. It originates from https://github.com/alphagov/govuk-developer-docs/commit/e3fae9b1cead4b39ddb41f3fc324cb8b6cf57341 - @chao-xian can you respond to this and [this](https://github.com/alphagov/govuk-developer-docs/pull/3966#discussion_r1172825462)?

Reply:
I think you might be better off without this section, because testing a Rake task from a development branch is no different to testing anything else from a development branch: deploy the branch, run the task.

Reply:
Removed

Reply:
that was from Leena's notes